README.md

```diff
@@ -45,7 +45,7 @@ Ensure packages are installed with correct version numbers by running:
 import 'react-dates/initialize';
 ```
 
-As of v13.0.0 of `react-dates`, this project relies on `react-with-styles`. If you want to continue using CSS stylesheets and classes, there is a little bit of extra set-up required to get things going. As such, you need to import `react-dates/initialize` to set up class names on our components. This import should go at the top of your application as you won't be able to import any `react-dates` components without it.
+As of v13.0.0, `react-dates` relies on [`react-with-styles`](https://github.com/airbnb/react-with-styles). Therefore, to get started, you need to import `react-dates/initialize` to set up class names on our components. This import should go at the top of your application as you won't be able to import any `react-dates` components without it.
```

Comment:
This advice isn’t the way to get started, it’s the way to get started *without react-with-styles* - using react-with-styles is the recommended approach.

Reply:
I don't know if that's a fair assessment @ljharb. The `initialize` script uses `react-with-styles` as well and it *is* the recommended out of the box approach.

Reply:
Is there another way to organize the `Initialize` section of the README then? More of a "do it this way with `react-with-styles`, otherwise you must do it this other way". I'm not familiar with `react-with-styles`, other than jumping over to that repo to briefly check out its purpose, so I'd imagine many others are not using it out of the gate either.